libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListenerRegistry.kt

```diff
@@ -48,6 +49,14 @@ internal class NavigationViewListenerRegistry(
 
     //endregion
 
+    fun onMapClicked(point: Point) {
+        for (listener in listeners.keys) {
+            if (listener.onMapClicked(point)) {
+                return
+            }
+        }
+    }
+
```

Comment:
Lets us remove this function to keep the implementation consistent. Refs: https://github.com/mapbox/mapbox-navigation-android/pull/6360#discussion_r976669830